### DISCUSSION TITLE
How can we implement the bug fix released for Microsoft.Azure.DurableTask.AzureStorage 1.9.3 inside Microsoft.Azure.WebJobs.Extensions.DurableTask

### DISCUSSION THREAD

**Discussion by Yangsh-w** [Azure/azure-functions-durable-extension#1964] [Q&A] [answered]
### Description
We reported an issue about an unexpected long delay between orchestrator 'schedule' and 'start', which turns out to be a known issue for **DurableTask.AzureStorage**, and the fix of this issue has been released in package [**Microsoft.Azure.DurableTask.AzureStorage 1.9.3**](https://www.nuget.org/packages/Microsoft.Azure.DurableTask.AzureStorage/).

We are using another package [**Microsoft.Azure.WebJobs.Extensions.DurableTask 2.5.1**](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.DurableTask/) in our project. This **DurableTask** package has **Microsoft.Azure.DurableTask.AzureStorage** as its dependency, so **AzureStorage** package should be installed automatically to our project I assume. 

But we could not find this AzureStorage package information(in .csproj or NuGet Management) for our project, so we are not sure how can we consume the fix for **Microsoft.Azure.DurableTask.AzureStorage 1.9.3** in our project.

### Expected behavior
The fix for **AzureStorage** got consumed by our project.

### Actual behavior
We are not sure which version of **AzureStorage** we are using and how we could consume the fix.

### App Details

- **Durable Functions extension version (e.g. v1.8.3)**: Microsoft.Azure.WebJobs.Extensions.DurableTask 2.5.1
- **Azure Functions runtime version (1.0 or 2.0)**: ~2.0 
- **Programming language used**: dotnet c#

**Comment by ConnorMcMahon** [comment, marked as answer]
Microsoft.Azure.WebJobs.Extensions.DurableTask v2.5.1 has a dependency on Microsoft.Azure.DurableTask.AzureStorage >= 1.9.0, so it will just pull in v1.9.0 by default.

If you explicitly add a PackageReference to Microsoft.Azure.DurableTask.AzureStorage v1.9.3, that will cause your app to build with v1.9.3 automatically.

`<PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.9.3" />`

**Comment by Yangsh-w** [comment]
Thank you very much @ConnorMcMahon ! I will test this solution.
